plugins/amazonq/codetransform/jetbrains-community/src/software/aws/toolkits/jetbrains/services/codemodernizer/CodeTransformTelemetryManager.kt

```diff
@@ -191,6 +191,15 @@ class CodeTransformTelemetryManager(private val project: Project) {
         )
     }
 
+    fun logHil(jobId: String, metaData: HilTelemetryMetaData) {
+        CodetransformTelemetry.humanInTheLoop(
+            project,
+            jobId,
+            metaData.toString(),
+            sessionId,
```

Comment:
I think result and reason should be bubble up like:

```
CodetransformTelemetry.humanInTheLoop(
            project,
            jobId,
            metaData.toString(),
            sessionId,
            reason: metaData.reason,
            result: metadata.result
            ```
            
We don't want to create new result/reason fields within our structure. What are your thoughts?